CorsixTH/Lua/app.lua

```diff
@@ -1827,15 +1827,16 @@ end
 function App:getSystemInfo()
   local compile_opts = TH.GetCompileOptions()
   local comp_details = {}
+  local date = os.date("%Y-%m-%d %H:%M:%S") -- Use ISO 8601 standard
```

Comment:
For what it's worth ISO 8501 requires a T not a ` `. Remove the misleading comment or fix the format. I have no strong preference.

Reply:
removed comment